apps/web/components/SearchInput/SearchInput.tsx

```diff
@@ -424,29 +426,46 @@ const Results: FC<{
               <Text variant="eyebrow" color="purple400">
                 {quickContentLabel}
               </Text>
-              {(search.results.items as Article[] &
-                LifeEventPage[] &
-                AboutPage[] &
-                News[])
+              {(search.results.items as
+                | (Article[] & LifeEventPage[] & AboutPage[] & News[])
+                | SubArticle[])
                 .slice(0, 5)
-                .map(({ id, title, slug, __typename }) => {
+                .map((item) => {
                   const { onClick, ...itemProps } = getItemProps({
                     item: '',
                   })
                   return (
                     <div
-                      key={id}
+                      key={item.id}
                       {...itemProps}
                       onClick={(e) => {
                         onClick(e)
                         onRouting()
                       }}
                     >
-                      <Link {...linkResolver(__typename as LinkType, [slug])}>
-                        <Text variant="h5" color="blue400">
-                          {title}
-                        </Text>
-                      </Link>
+                      {item.parentSlug ? (
```

Comment:
Note: Here I am testing how the data gets delivered to the front.
I highly suspect this will be changed by someone who knows the frontend well